ecs-cluster/values.yaml

```
@@ -123,10 +123,12 @@ pravega:
 #    replicas: 1
     service:
       type: ClusterIP
+    disableAntiAffinity: false
```

why not be positive, it sounds backwards to have disable false double negative.

```suggestion
    enableAntiAffinity: true
```